simpl/db/mongodb.py

```diff
@@ -115,6 +115,55 @@ def scrub(data):
         raise ValidationError("Input '%s' not permitted: %s" % (data, exc))
 
 
+def build_text_search(strings, name_field='name'):
```

Comment:
Was just looking over the `$text` documentation, seems like it would be a nice feature and pretty easy to add negations to your string, specified by another keyword argument on this function?

http://docs.mongodb.org/master/reference/operator/query/text/#exclude-documents-that-contain-a-term